.github/workflows/pr-review-client-deploy.yml

```diff
@@ -0,0 +1,145 @@
+          aws ecr batch-delete-image \
+            --repository-name $IMAGE \
+            --image-ids "$IMAGES_TO_DELETE" || true
+
+      - name: Logout of Amazon ECR
+        run: docker logout $REGISTRY
+
+  deploy-test-client:
+    if: ${{ !contains(github.event.pull_request.labels.*.name, 'Renovate') }}
+    runs-on: ubuntu-latest
+    needs: build-and-push-container
+    steps:
+      - name: Set envs
+        run: echo "PR_NUMBER=$(jq --raw-output .pull_request.number "$GITHUB_EVENT_PATH")" >> $GITHUB_ENV
+
+      - name: Configure AWS credentials
+        id: aws-creds
+        uses: aws-actions/configure-aws-credentials@67fbcbb121271f7775d2e7715933280b06314838 # v1.7.0
+        with:
+          aws-access-key-id: ${{ secrets.AWS_ACCESS_KEY_ID }}
+          aws-secret-access-key: ${{ secrets.AWS_SECRET_ACCESS_KEY }}
+          aws-region: ca-central-1
+
+      - name: Create/Update lambda function
+        run: |
+          if aws lambda get-function --function-name $FUNCTION_NAME-$PR_NUMBER > /dev/null 2>&1; then
+            aws lambda update-function-code \
+              --function-name $FUNCTION_NAME-$PR_NUMBER \
+              --image-uri $REGISTRY/$IMAGE:$PR_NUMBER > /dev/null 2>&1
+
+          else
+            aws lambda create-function \
+              --function-name $FUNCTION_NAME-$PR_NUMBER \
+              --package-type Image \
+              --role $ROLE_ARN \
+              --timeout 15 \
+              --memory-size 1024 \
+              --code ImageUri=$REGISTRY/$IMAGE:$PR_NUMBER \
+              --description "$GITHUB_REPOSITORY/pull/$PR_NUMBER" \
+              --vpc-config SubnetIds=${{ secrets.PR_REVIEW_ENV_SUBNET_IDS }},SecurityGroupIds=${{ secrets.PR_REVIEW_ENV_SECURITY_GROUP_IDS }} > /dev/null 2>&1
+
+            aws lambda wait function-active --function-name $FUNCTION_NAME-$PR_NUMBER
+            aws lambda add-permission \
+              --function-name $FUNCTION_NAME-$PR_NUMBER \
+              --statement-id FunctionURLAllowPublicAccess \
+              --action lambda:InvokeFunctionUrl \
+              --principal "*" \
+              --function-url-auth-type NONE > /dev/null 2>&1
+
+            URL="$(aws lambda create-function-url-config --function-name $FUNCTION_NAME-$PR_NUMBER --auth-type NONE | jq .FunctionUrl)"
+            echo "URL=$URL" >> $GITHUB_ENV
+
+            aws lambda update-function-configuration \
+              --function-name $FUNCTION_NAME-$PR_NUMBER \
+              --environment "Variables={\
+                NOTIFY_ENVIRONMENT=staging,\
+                FLASK_APP=application.py,\
+                IP_GEOLOCATE_SERVICE=False,\
+                REDIS_ENABLED=True,\
+                SENDING_DOMAIN=staging.notification.cdssandbox.xyz,\
+                API_HOST_NAME=https://api.staging.notification.cdssandbox.xyz,\
+                ADMIN_BASE_URL=$URL
```

Comment:
We'll need to figure out what values are needed here.   It's going to be something similar to how the ECS task is configured but we can figure it all out through clickops first.